client/components/push-notifications.jsx

```diff
@@ -171,7 +171,7 @@ export default class PushNotifications extends React.Component {
 
       <h3>Push Notification Subscription</h3>
       <code>
-        {JSON.stringify(this.state.subscription, null, 4)}
+        {JSON.stringify(this.state.subscription)}
```

Comment:
This string is actually rendered to the screen, the curl command is also rendered below it. It wasn't very pretty though, it might be better to just do something like:

```jsx
<h3> Endpoint </h3>
<code>
{this.state.subscription.endpoint}
</code>
```